apps/wallet-mobile/src/yoroi-wallets/cardano/shelley/ShelleyWallet.ts

```diff
@@ -158,9 +155,11 @@ export const makeShelleyWallet = (constants: typeof MAINNET | typeof TESTNET) =>
   }
 
   const appApi = AppApi.appApiMaker({baseUrl: API_ROOT})
+  const cardanoApi = CardanoApi.cardanoApiMaker({baseUrl: API_ROOT_NEW})
```

Comment:
Are the rest of cardano apis work with the new api root?

Reply:
@banklesss the new api root is used only for the `protocol params`. Later when the new backend is ready we will use only this `new api` and remove the old one

Reply:
- yoroi-backend: the one Yoroi always used. Legacy, bad and to be completely replaced by yoroi-backend-zero;

- yoroi-backend-zero: new backend that will replace yoroi-backend, but right now you will use only the /protocolparameters endpoint of it;

Reply:
tenemos que considerar eso si queremos extender cardanoApi 

cc @stackchain

Reply:
the apiMaker should receive the `network` only, and the endpoints should be part of the module, not the wallet.